filter.go

```diff
@@ -24,6 +24,7 @@ type Filter struct {
 	EventFormat EventFormat `json:"event_format,omitempty"`
 	Presence    FilterPart  `json:"presence,omitempty"`
 	Room        RoomFilter  `json:"room,omitempty"`
+	ToDevice    *FilterPart `json:"com.beeper.to_device,omitempty"`
```

Comment:
The Go field name should also be prefixed

```suggestion

	BeeperToDevice *FilterPart `json:"com.beeper.to_device,omitempty"`
```